src/components/badges/BadgeSection.tsx

```diff
@@ -15,7 +15,7 @@ const badgesList = classnames(
   display('grid'),
   gap('gap-2'),
   gridAutoRows('auto-rows-auto'),
-  gridTemplateColumns('grid-cols-1', 'md:grid-cols-2', 'lg:grid-cols-3')
+  gridTemplateColumns('grid-cols-1', 'sm:grid-cols-2', 'lg:!grid-cols-6')
```

Comment:
Are you sure that `grid-cols-2` should be on `sm` screens, and not on `md`?

Reply:
Yes, it looks just fine on 450px-1024px:
![image](https://user-images.githubusercontent.com/49658988/187437722-162a66a3-4bde-455b-8e70-0f46b2fa69d9.png)
![image](https://user-images.githubusercontent.com/49658988/187437798-334c892f-5daa-466b-8b40-f30691def6a4.png)

Reply:
Except 600-700, but it's a layout problem, I think, I'll fix it
![image](https://user-images.githubusercontent.com/49658988/187437991-19dcd0ea-b18e-4f62-a62c-546b0baff471.png)